test/index.js

```diff
@@ -28,12 +28,12 @@ const runTest = function(url) {
         error: '>=2000'
       },
       psi: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=2000',
+        error: '>=3500'
       },
       vc85: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=3500',
+        error: '>=5000'
```

Comment:
Would it make sense to also only access the discourse API at build time, just like you do with github? Since I guess the version-links from both services will probably/hopefully only be changing at the same time.